Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+RUN mkdir -p /app
+
+COPY . main.py /app/
```

Comment:
you can run "COPY . /app/" to copy all files (including requirements.txt) to our work directory /app.

Reply:
I've done that.